main.js

```diff
@@ -0,0 +1,85 @@
+
+
+console.log("you are here");
+
+
+// create variables needed
+var count;
+var row;
+var container;
+var expAmt;
+var expName;
+var addExpAmt;
+var addUpExp;
+var removeExp = document.querySelector('span');
```

Comment:
You're going to have more than one `removeExp`, actually one for each row, therefore I don't think you'd want to query the DOM for it at this point.